TeachingRecordSystem/src/TeachingRecordSystem.Core/Services/Files/ServiceCollectionExtensions.cs

```diff
@@ -7,6 +8,7 @@ public static class ServiceCollectionExtensions
     public static IServiceCollection AddFileService(this IServiceCollection services)
     {
         services.AddSingleton<IFileService, BlobStorageFileService>();
+        services.AddSingleton<IDqtNoteAttachment, DqtNoteAttachment>();
```

Comment:
This shouldn't be in this method; add a new `AddBlobStorageDqtNoteAttachments` extension method in the `Core.Services.DqtNoteAttachments` namespace.